modules/netns-isolation.nix

```diff
@@ -293,6 +293,17 @@ in {
       clightning-rest = {
         id = 30;
       };
+      fulcrum = {
+        id = 31;
+        connections = [ "bitcoind" ];
+      };
+      mysql = {
+        id = 32;
+      };
+      mempool = {
+        id = 33;
+        connections = [ "bitcoind" "electrs" "fulcrum" "nginx" "mysql" ];
```

Comment:
Connections to `mysql` netns don't need to be allowed since we're now using a UNIX socket